wlm_server/task/handler.py

```diff
@@ -21,12 +24,58 @@ class TaskHandler(threading.Thread):
 
     def __init__(self):
         super().__init__()
+        self._wlm: WLM
         self._message_queue: MessageQueue = settings.MESSAGE_QUEUE
         self._measure_queue: MeasureQueue = MeasureQueue()
         self._channel_to_period: dict[int, timedelta] = {}
+        self._open_connection()
+
+    def _open_connection(self):
```

Comment:
This method may cause several errors, e.g., there is no `Config`, failed to open WLM.
In fact, regarding WLM, it is always possible to fail the API call as the USB connection goes wrong sometimes.
I think we should carefully handle such error cases.

Reply:
Maybe modeling this handler as an FSM would be helpful..?

Reply:
Umm.. Indeed, I didn't consider any error on purpose.
To send the occurred error info to the request handler, we need to implement more APIs, hence I'd like to handle this after implementing main features.

What do you think about this? @kangz12345